tests/test_streams.py

```diff
@@ -122,5 +122,20 @@ def test_eeg_channel_count():
         assert window.shape[0] == 16  # Total number of eeg channels for synthetic board
 
 
+def test_ring_buffer_shift():
```

Comment:
Is this code needed? I see you've removed the ring buffer computation from main.py

Reply:
Old version, this no longer exists.